test/mlir/conversion/onnx_to_krnl/Sequence/Sequence_with_dealloc.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: onnx-mlir-opt -O3 --shape-inference --convert-onnx-to-krnl --buffer-deallocation-pipeline  %s -split-input-file | FileCheck %s
```

Comment:
@tungld Do you know why this lit test had a pass named `--buffer-deallocation-pipeline ` when I only see `--buffer-deallocation` pass defined... so I changed it back. Let me know your thoughts please if there is any other changes I should make. Thanks!

Reply:
I guess this test is to check if `memref.dealloc` is inserted correctly or not. When you changed it back, I saw that `memref.dealloc` was missing.

I think `--buffer-deallocation` is the old mechanism, so we moved to the new one `--buffer-deallocation-pipeline` from the PR https://github.com/onnx/onnx-mlir/pull/2568.
It's not recommended to go back to the old mechanism since it will finally be removed in LLVM. 

When I blamed this file, @chentong319 is the original author introducing this test. @chentong319 may have a better explanation.